.circleci/config.yml

```diff
@@ -652,6 +652,18 @@ jobs:
       - DEPLOY_DOWNTIME: 1800 #30M
     steps: *deploy_steps
 
+  deploy_next:
+    <<: *infrastructure_config
+    environment:
+      - DEPLOY_ENV: next
+    steps: *deploy_steps
+
+  deploy_unstable:
+    <<: *infrastructure_config
+    environment:
+      - DEPLOY_ENV: unstable
```

Comment:
Not sure if any downtime needed.

Reply:
They are just 3 nodes, so i'll give them 15 min